nodes/packet-arm64-beta.json

```diff
@@ -0,0 +1,11 @@
+{
+  "run_list": [
+    "role[jenkins-slave-arm-bionic]"
```

Comment:
I am not terribly certain just now, but I think you can have

```
 "role[jenkins-slave-arm]",
 "role[ruby-bionic]"
```

where the latter defines the override_attributes for ruby build. Chef will assemble the entire attribute set before running anything.

(Technically it'd be much nicer to have a new recipe added to the the server-common role which adjusts install_pkgs_cruby problematically based on the platform version so we don't need to explicitly specify anything but the cookbooks figure things out themselves; not sure if you wanna start a whole new cookbook though ;))

Reply:
I am leaving this as-is for now. Will fix it in different PR.